### PR TITLE
Automated cross browser testing

### PR DESCRIPTION
**Pull request by saedwards**
### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

**Comment by saedwards** [comment]
> @saedwards looking good - all tests passed 👍 what's next?

I'm configuring Karma locally but run into usage problems. Browser Stack is giving me the following error with our paid license: 'Error: Your free access to test on real devices has expired. You can upgrade to the Automate Mobile plan to test on real devices. Please reach out to our Sales team at sales@browserstack.com.'

I've contacted them regarding this.

**Comment by saedwards** [comment]
After Karma is configured with the right browsers, we be looking to increase our code coverage, otherwise we'll still get problems. Each developer should write tests for their code (old and new components).

File: tests/karma/karma.conf.local-launchers.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ export default function () {
 
   return {
     browsers: [
-      //'Chrome',
+      'Chrome',
       'Firefox',
-      //'HeadlessChrome'
+      'HeadlessChrome'
```
Comment (bameyrick) [inline tests/karma/karma.conf.local-launchers.js]: What is the benefit of running headless chrome as well as standard chrome?

File: assets/js/api/_load.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
```
Comment (bameyrick) [inline assets/js/api/_load.js]: This is also imported in `components.js` which is bundled with this file in the gulp build task. Could this result in some duplication in `bundle.js` even with uglify running?

File: assets/js/inpagelink.js [hunk 3]
```diff
@@ -1,32 +1,35 @@
...
 
 export default function() {
-  return inPageLink()
+  return inPageLink();
 }
 
 export function inPageLink() {
-  const nodeList = document.getElementsByClassName(classTrigger)
-  forEach(nodeList, applyInPageLink)
+  const nodeList = document.getElementsByClassName(classTrigger);
```
Comment (bameyrick) [inline assets/js/inpagelink.js]: The lodash dependency can be removed by casting the node list to an array like so which will allow you to use native forEach

```suggestion
  const nodeList = [...document.getElementsByClassName(classTrigger)];
```

File: tests/karma/karma.conf.local-launchers.js (hunk 1 above)
Comment (saedwards) [reply tests/karma/karma.conf.local-launchers.js]: Part of previous configuration. I haven't chosen to remove it.

File: assets/js/api/_load.js (hunk 2 above)
Comment (saedwards) [reply assets/js/api/_load.js]: Not related to changes in the context of this PR, create an issue?

File: assets/js/inpagelink.js (hunk 3 above)
Comment (saedwards) [reply assets/js/inpagelink.js]: Not related to changes in the context of this PR, create an issue?

**Pull request merged**